osu.Game/Skinning/LegacyOsuComboCounter.cs

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Framework.Threading;
+using osu.Game.Screens.Play.HUD;
+using osuTK;
+
+namespace osu.Game.Skinning
+{
+    /// <summary>
+    /// Uses the 'x' symbol and has a pop-out effect while rolling over.
+    /// </summary>
+    // todo: this shouldn't be here, hmm...
+    public class LegacyOsuComboCounter : LegacyComboCounter
```

Comment:
How about removing this TODO because it's probably not going away. Maybe rename the class to `LegacyDefaultComboCounter`?

Reply:
Ah, meant this one for myself, forgot to remove it before PR.